### PR TITLE
73 fix code smells in chatview and chatrenderusecase

### PR DESCRIPTION
**Pull request by czhang0312**
Fixed smelly code in chatView and cleaned up intellij warnings

### CODE REVIEW

File: src/main/java/FrameworksDrivers/ChatView.java [hunk 1]
```diff
@@ -162,64 +160,13 @@ public void updatePage(Object[] info) {
                 currChatPanel.setBackground(Color.CYAN);
 
                 //iterate through the messages of the current chatroom
-                for (int i = 2; i < chatroomList.size(); i++) {
```
Comment (czhang0312) [inline src/main/java/FrameworksDrivers/ChatView.java:165]: I broke down the huge for loop into a bunch of helper functions

Comment (skoooch) [reply src/main/java/FrameworksDrivers/ChatView.java:165]: Very Swag!

**Review by skoooch — DISMISSED** [review]
LGTM!

**Review by vvickychan — DISMISSED** [review]
The edits and tests look good to me

**Review by skoooch — APPROVED** [review]
LOoks dope!

**Review by vvickychan — APPROVED** [review]
Comments were left previously

**Review by skoooch — APPROVED** [review]
lgtm

**Review by jay-ead — APPROVED** [review]
This code smells fantastic now 👍

**Pull request merged**